### ISSUE TITLE
Automatically and persistently forcing JSX to indent in a non-preferred way.

### ISSUE THREAD

**Issue by mliq** [gandm/language-babel#237]
2.35.0 has caused an issue for me.

JSX is automatically indented in the following way:

```
<Component prop1="stuff"
prop2="stuff">
</Component>
```

Whereas previously, (and as required by my project's eslint rules), I had:

```
<Component prop1="stuff"
           prop2="stuff">
</Component>
```

I tried toggling off all settings. 
This unwanted indentation change occurs whenever I click on a line. And, if I try to change the indentation, it immediately sets it back again to the unwanted version.

For now, my workaround was:

```
apm rm language-babel
apm install language-babel@2.34.0
```

OSX 10.11.6
Atom 1.9.9

Thanks!

**Comment by gandm** [comment]
The change in 2.35.0 was very minor and only affected a simple grammar regex for // comments. I have just tried it and it seems fine. Can you try reinstalling 2.35.0, sometimes the Atom cache gets out of step and causes strange issues.

**Comment by mliq** [comment]
@gandm You are right, now it works, thanks!

**Comment by mliq** [comment]
@gandm This happened to me again, I think I fixed it by erasing the `"language-babel"` section in my config.cson.

For reference it was:
```
"language-babel":
    allowLocalOverride: true
    autoIndentJSX: true
    babelMapsAddUrl: false
    createTargetDirectories: false
    suppressSourcePathMessages: false
    suppressTranspileOnSaveMessages: false
```

**Comment by gandm** [comment]
I can't see why the config.cson that would be an issue. Are you sure it wasn't because atom was updated to 1.10.0 recently?

**Comment by gandm** [comment]
Or maybe it was language-Babel updating a few times today?

**Comment by mliq** [comment]
@gandm It could be the Atom update that re-triggered my issue initially, and/or the language-babel updates. However, it started after all updates for the day had completed and I had restarted Atom, and all of the actions I took to try and fix the problem occurred with no intervening updates.

Those actions included:
```
apm rm language-babel
apm install language-babel
```
(this had fixed my problem initially on 1.9.9)

and the same but with: `apm install language-babel@2.36.0` (the last known working version)

but nothing fixed it until the config.cson change.
